localtypings/pxtarget.d.ts

```diff
@@ -164,6 +164,7 @@ declare namespace pxt {
         invertedMonaco?: boolean; // if true: use the vs-dark monaco theme
         blocklyOptions?: Blockly.Options; // Blockly options, see Configuration: https://developers.google.com/blockly/guides/get-started/web
         disableBlockIcons?: boolean; // Disable icons in blocks
+        flyoutHeadings?: boolean; // Show headings at the top of the flyout
```

Comment:
this should be always on -- with option to turn it off ( ``flyoutHeadings`` -> ``hideFlyoutHeadings``)